requirements/dev-requirements.in

```diff
@@ -2,7 +2,8 @@
 nbdev
 pre-commit
 black
-nb-black
+# nb-black (outdated install requirements, can not be installed)
```

Comment:
Black comes with Jupyter support these days:
`black`
can be changed to
`black[jupyter]`

https://black.readthedocs.io/en/stable/getting_started.html#installation

Reply:
changed to black, removed nb-black completely